apps/devhub/tests/test_views.py

```diff
@@ -2539,9 +2539,9 @@ def xpi(self):
                     'rb')
 
     @mock.patch.object(waffle, 'flag_is_active')
+    @mock.patch('devhub.tasks.validate')
```

Comment:
Those two tests are weird... `raise AssertionError`? Why not `assert ...`.
Anyway, I don't understand why you mock `validate`, but still call it manually on line 2560? Is the `run_validator` mock still needed?
I feel that one of those two mocks isn't needed.